src/components/AppNavigationCaption/AppNavigationCaption.vue

```diff
@@ -7,6 +7,23 @@
 	</template>
 </AppNavigationCaption>
 ```
+
+### Element with a slot for custom actions icon
+```
+<AppNavigationCaption
+	title="Your caption goes here">
+	<template #actionsTriggerIcon>
+		<PlusIcon slot="icon" :size="20" />
```

Comment:
The `PlusIcon` component is missing and not showing. You need to import it. Have a look at e.g. https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/Actions/Actions.vue#L88-L122 for how to.

Reply:
ok, something went wrong =(
actually i thought, that it is already imported.
Thank you!